fuzzers/nyx_libxml2_standalone/src/main.rs

```diff
@@ -10,6 +10,7 @@ use libafl::{
     feedbacks::{CrashFeedback, MaxMapFeedback},
     inputs::BytesInput,
     monitors::tui::TuiMonitor,
+    monitors::tui::ui::TuiUI,
```

Comment:
can you run cargo fmt

Reply:
Yes, I have run `cargo fmt` before I commit this PR.

Reply:
You need to run it both in the project's top directory and the directory of the fuzzer that you changed